scripts/generate_parity_inventory.py

```
@@ -507,16 +510,16 @@ def build_inventory() -> dict:
         "schema_version": "parity-inventory-v2",
         "generated_by": "scripts/generate_parity_inventory.py",
         "counts": {
-            "pandas_diagnostics": 59,
+            "pandas_diagnostics": 62,
             "sql_analytics": 4,
-            "sql_registry": 63,
+            "sql_registry": 66,
             "concepts": len(concepts),
             "aliases": len(aliases_index),
-            "building_100_cartesian": "48 equipment × 59 diagnostics = 2832 results",
+            "building_100_cartesian": "48 equipment × 62 diagnostics",
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)